src/utils.js

```diff
@@ -126,14 +127,23 @@ export function getVariable(context, name) {
  *
  */
 export function gettext(str) {
-  return str;
+  const locale = process.env.LANG || process.env.LANGUAGE || process.env.LC_ALL || process.env.LC_MESSAGES || 'en-US';
```

Comment:
Let's move this into a utility function, e.g something called `getLocale` so that it's easier to re-use later if needed.